timetracker/entries/admin.py

```diff
@@ -3,10 +3,25 @@
 # Register your models here.
 from . import models
 
-class EntryAdmin(admin.ModelAdmin):
-    list_display = ('project', 'description', 'start', 'stop')
-    list_filter = ('project',)
-    search_fields = ('project', 'description',)
+# Django admin site doesn't create forms using attributes from differnet models, so each model must be edited seperately (!?)
```

Comment:
Nice pick up! I think we'll cover it in Lesson 6 (Advanced Topics), but if you're interested you should check out the documentation on Admin Inlines. You create them in a _very_ similar way to what you're done here.

https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#inlinemodeladmin-objects